src/schema/comments.ts

```diff
@@ -618,7 +608,10 @@ export const resolvers: IResolvers<any, BaseContext> = {
               .andWhere(`${builder.alias}.postId = :postId`, {
                 postId: args.postId,
               })
-              .andWhere(`${builder.alias}.parentId is null`);
+              .andWhere(`${builder.alias}.parentId is null`)
+              // Only show comments that vordr prevented, if the user is the author of the comment
+              .andWhere(whereVordrFilter(builder.alias, ctx.userId))
```

Comment:
We have `commentFeed` do we want to apply it there as well?

Reply:
Where's that one used?

Reply:
https://app.daily.dev/discussed

Reply:
Ah, yeah. I'll add it there, and probably on the Replies section on user tab.

Reply:
Is it fine that I do it in a separate PR, want to be sure the filter works as expected in production

Reply:
sure

Reply:
👍